### PR TITLE
ci: publish

### PR DESCRIPTION
**Pull request by Odrin**
## Status

**READY**

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

### CODE REVIEW

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v6.1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          custom_tag: ${{ steps.yaml_data.outputs.data }}
+
+      - name: Create GitHub release
+        uses: softprops/action-gh-release@v1
+        with:
+          tag_name: ${{ steps.tag_version.outputs.new_tag }}
```
Comment (nesquikm) [inline .github/workflows/publish.yaml]: U miss new line at eof.

File: .github/workflows/publish.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+name: Publish
+
+on:
+  pull_request:
+    types: [closed]
```
Comment (nesquikm) [inline .github/workflows/publish.yaml:5]: Is it means that version should be created even if the PR vas closed and not merged?

Comment (Odrin) [reply .github/workflows/publish.yaml:5]: We have conditional job execution:
```
if: github.event.pull_request.merged && github.head_ref == 'version-bump'
```

Comment (nesquikm) [reply .github/workflows/publish.yaml:5]: Good, thnx!

**Review by nesquikm — APPROVED** [review]

**Pull request merged**